### ISSUE TITLE
docs: add deprecation info for properties, methods, etc.

### ISSUE THREAD

**Issue by jenniferfell** [angular/angular#30098]
# 📚 Docs or angular.io bug report

### Description
Currently, we mark exports as deprecated in the API reference docs and mention them in the Deprecation Summary. 

We do not mention properties, methods, etc. consistently in the  Deprecation Summary. They are, however, marked `@deprecated` in the source code. When they are marked in code, they do not appear in the API reference. 

@trotyl makes the reasonable recommendation that we do more to highlight these deprecations as well. See https://github.com/angular/angular/pull/26733#discussion_r278041763

 A clear and concise description of the problem...

## 🔬 Minimal Reproduction
Some examples: 

```
WrappedValue#wrapped (code) --example of omitting from API ref
IterableDiffers#factories (code)
KeyValueDiffers#factories (code)
one of Injector#get() overload (code)
one of Injector.create() overload (code) -- example of text saying deprecated in API ref
ViewContainerRef#parentInjector (code)
Testability#increasePendingRequestCount (code)
Testability#decreasePendingRequestCount (code)
```

Same for other packages, can be found by searching @deprecated in source code.

### What's the affected URL?**
 https://angular.io/...
https://angular.io/api
https://angular.io/api/*
https://next.angular.io/guide/deprecation (soon to be https://angular.io/guide/deprecations)

### Reproduction Steps**

See above

### Expected vs Actual Behavior**

See above

## 🔥 Exception or Error
<pre><code>

</code></pre>

**Comment by ajitsinghkaler** [comment]
@gkalpak @kapunahelewong I think we can close this I resolved the issue

**Comment by gkalpak** [comment]
@ajitsinghkaler, can you please add more info on how/where this issue was resolved?

**Comment by ajitsinghkaler** [comment]
@gkalpak I resolved it in #34088
